src/Esfa.Vacancy.Register.Application/Queries/SearchApprenticeshipVacancies/StandardCodeConverter.cs

```diff
@@ -34,7 +35,9 @@ public StandardCodeConverter(IStandardRepository standardRepository)
                 var standardSector = standardSectorIds.FirstOrDefault(ss => ss.LarsCode == parsedStandardToConvert);
                 if (standardSector == null)
                 {
-                    result.ValidationFailures.Add(new ValidationFailure("StandardCode", $"StandardCode {parsedStandardToConvert} is invalid"));
+                    result.ValidationFailures
+                        .Add(new ValidationFailure("StandardCode", string.Format(ErrorMessages.SearchApprenticeships.StandardCodeNotFound, parsedStandardToConvert))
```

Comment:
Rather than use the constructor, could we not use object initialization since there isn't a constructor that allows Error code and I don't think whatever the constructor is, is doing anything special? Or is this the only constructor on that class?

Reply:
Valid question, however just a note, this class is being deleted as part of what I am doing.